src/main/webapp/js/instructorHome.js

```diff
@@ -1,27 +1,61 @@
+var COURSE_PANELS_TO_AUTO_LOAD_COUNT = 3;
+
 $(document).ready(function() {
     setupFsCopyModal();
     
     //Click event binding for radio buttons
-    var radiobuttons = $("label[name='sortby']");
-    $.each(radiobuttons, function() {
+    var radioButtons = $('label[name="sortby"]');
```

Comment:
Refrain from refactoring unrelated code. Anyway this should be prefixed with a $